src/clients/java/pom.xml

```diff
@@ -243,6 +243,7 @@
                 <argument>build</argument>
                 <argument>java_client</argument>
                 <argument>-Doptimize=ReleaseSafe</argument>
+                <argument>-Dconfig=production</argument>
```

Comment:
Such a great find!

Reply:
It was with @kassane 's help!
Thanks for reporting the bug.